SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2972,6 +2987,21 @@ void USpatialNetDriver::TryFinishStartup()
 			Connection->SetStartupComplete();
 		}
 
+		if (WorkerType == SpatialConstants::StrategyWorkerType)
+		{
+			SpatialGDK::FSubView& NewView =
+				Connection->GetCoordinator().CreateSubView(SpatialConstants::STRATEGYWORKER_TAG_COMPONENT_ID,
+														   [](const Worker_EntityId, const SpatialGDK::EntityViewElement&) {
+															   return true;
+														   },
+														   {});
+
+			StrategySystem = MakeUnique<SpatialGDK::SpatialStrategySystem>(NewView, Connection->GetWorkerSystemEntityId());
+			StrategySystem->Init(Connection);
```

Comment:
Can we maybe add `Connection` as a constructor argument? Right now, it seems that we have to always call `Init` before the `StrategySystem` is actually complete.

Reply:
This follows the same pattern as the RoutingSystem is there a reason @ImprobableNic that it was not done in the constructor?

Reply:
The Init part is inherited from how UObject are setup, although none of these systems are.
So no particular reason as to why we need a separate Init method, this can be changed.

Reply:
Thanks, I've merged the Init method into the constructor.